workers/unity/Packages/io.improbable.gdk.core/Tests/Editmode/Connection/CommandLineConnectionParametersInitializerTests.cs

```diff
@@ -30,7 +30,6 @@ public void Initialize_should_set_network_protocol(string protocolStr, NetworkCo
 
         [TestCase("ranket")]
         [TestCase("raknet")]
-        [TestCase("")]
```

Comment:
Had to remove this. The initializer now assumes empty string is a result of parsing a flag argument and will throw `InvalidOperationException` when you try to use `TryGetCommandLineValue` on it.